_includes/comment-form.html

```diff
@@ -0,0 +1,96 @@
+
+{% if site.get-form-action-url %}
+    {% if site.g-re-captcha-site-key %}
+    <script>
+        (function () {
+            var d = document, s = d.createElement('script');
+            s.src = 'https://www.google.com/recaptcha/api.js';
+            (d.head || d.body).appendChild(s);
+        })();
+    </script>
+    <script>
+        function onFocusOut(element) {
```

Comment:
a cosa serve questa function?

Perche' aggiungi la classe `required-input` al focusout del div?

Reply:
Perché se la aggiungo a priori me li da non validi all'apertura (perché sono vuoti all'inizio). In questo modo devi prima entrarci per far partire la validazione